xxxx-template.md

```diff
@@ -0,0 +1,43 @@
+- Start Date: (fill in today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+
+# Summary
+
+Description of the feature or changes.
+
+# Example
+
+If the proposal requires changes to the current API or the creation of new ones, add a basic code example.
+
+# Motivation
+
+Please make sure to explain the motivation for this proposal. 
+It means explaining the usecases and the fonctional features this proposal should be solving. 
```

Comment:
It means explaining the use case(s) and the functional feature(s) this proposal is trying to solve.